### ISSUE TITLE
Bluefruit.begin() and USB Device Not recognized. 

### ISSUE THREAD

**Issue by innoagg** [adafruit/Adafruit_nRF52_Arduino#544]
Compiling with the current master branch the USB device is not recognized. An empty sketch calling Bluefruit.begin() in the setup is all that is required to reproduce. 

```
#include <stdio.h>
#include <Arduino.h>
#include <bluefruit.h>

void setup() {
  Serial.begin(115200);
  Bluefruit.begin(0, 4);
}

void loop() {
  
}

```

Boards: pca10059, pca10056
Windows 10

![image](https://user-images.githubusercontent.com/11233095/88682052-3afa4580-d0c0-11ea-8bfd-e8e56ae21132.png)

**Comment by hathach** [comment]
I confirmed the issue, there is still issue with portYield() in ISR https://github.com/hathach/tinyusb/pull/468 and the hardware initialization. Initialize it late cause missing initial setup packet. I am troubleshooting it now.

**Comment by hathach** [comment]
This really give me a headache now, I need to do commit bisecting to figure the reason.

**Comment by innoagg** [comment]
Wow, thank you for all your efforts. Much appreciated. Try to enjoy your weekend!

**Comment by hathach** [comment]
should be fixed by #546 but need https://github.com/hathach/tinyusb/pull/478 to be merged first
